File: Netbeans/Java/Lab6/test/model/SumAndQuantityTest.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

import org.junit.Before;
import org.junit.Test;
import static org.junit.Assert.*;

/**
 *
 * @author prostrmk
 */
public class SumAndQuantityTest {
    
    public SumAndQuantityTest() {
    }
    
    @Test
    public void quantity(){
        int n = 13;
        assertEquals(2, SumAndQuantity.quantity(n));
    }

    @Test
    public void sumTest() {
        int n = 13;
        assertEquals(4, SumAndQuantity.sum(n));
    }
    
}

File: Csharp/Laba6/Laba6/TestFolder/SumAndQuantityTest.cs
using NUnit.Framework;
using System;

namespace Laba6
{
	[TestFixture ()]
	public class SumAndQuantityTest
	{
		[Test ()]
		public void SumTester ()
		{
			int sum = 45;
			Assert.AreEqual (9, SumAndQuantity.Sum (sum));
		}
		public void Quantity()
		{
			int qu = 45;
			Assert.AreEqual (2, SumAndQuantity.Quantity (qu));
		}
	}
}

